src/libs/elektra/libelektra-symbols.map

```diff
@@ -5,6 +5,8 @@
 		kdb*;
 		key*;
 		ks*;
+		KDB*;
+		_elektra*;
```

Comment:
We should avoid exporting functions starting with _.

Reply:
The _elektra* functions are needed for the code generation. The prefix is meant to indicate that these functions are not intended for manual use.

If you don't like the _elektra, we could use elektraInternal* or something similar that indicates the internal nature of the function.

Reply:
Yes, functions starting with _ might be reserved. elektraGen or elektraInternal should do.

Why is KDB needed?

Reply:
>Why is KDB needed?

Because of the `KDB_TYPE_*` constants. These are also used in code generation, but are useful for the end user as well, e.g. if used with `elektraSetValue`.